app/active/big-brother-26/league-standing/page.tsx

```diff
@@ -0,0 +1,24 @@
+import { WIKI_API_URL } from '../../../leagueConfiguration/BigBrother_26'
+import { CONTESTANT_LEAGUE_DATA } from '../../../leagueData/BigBrother_26'
+import { getWikipediaContestantDataFetcher } from '../../../utils/wikiFetch'
+import { getCompetingEntityList } from "../../../utils/wikiQuery"
+import generateListOfContestantRoundLists from '../../../generators/contestantRoundListGenerator'
+import LeagueStandingTable from '../../../components/leagueStandingTable/leagueStandingTable';
+
+export default async function LeagueStanding() {
+    const dataFetcher = getWikipediaContestantDataFetcher(WIKI_API_URL, "HouseGuests")
+    const contestantScoresData = await generateListOfContestantRoundLists(dataFetcher, CONTESTANT_LEAGUE_DATA, getCompetingEntityList)
```

Comment:
How come we don't use `generateContestantRoundScores`?

Reply:
because of the requirement of `team`.  I figured I'll refactor in the follow up ticket when I refactor all of that crazy logic

Reply:
Resolved in a cleanup. Marking as resolved due to the outdated nature of the code